### DISCUSSION TITLE
Cannot see Request Access buttons to Copilot for my Org

### DISCUSSION THREAD

**Discussion by dunnapey** [community/community#129240] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I am signed into my org that has enabled Copilot access but I was told there is supposed to be ASK ADMIN FOR ACCESS buttons next to the org in GitHub settings, but I do not see those buttons & cannot request access. Is this a bug on GitHub or is there something on my org's end that needs to be fixed to enable the request buttons?
What I see:
![image](https://github.com/community/community/assets/22160417/8fda90fb-4f51-45e0-973f-c914b4934d48)
What my org sees:
<img width="914" alt="image" src="https://github.com/community/community/assets/22160417/7fd42671-2d99-422d-a262-c5d0b4fe48a7">

**Comment by dunnapey** [comment]
It would appear I have an individual subscription active from when I was a verified student, but seems like it prohibits me from joining my org's subscription. Do I have to wait till that expires before I can join my org's plan or is there a way to cancel my individual plan? My org's plan has safeguards for our enterprise code that individual plan doesn't honor, so I'd like to switch if possible.

**Reply by ayim** [reply]
@Akash1134 could you help us route this to the product team that worked on this feature?

**Reply by dunnapey** [reply, marked as answer]
FYI: My org admin was able to add me manually, & GitHub now shows my Copilot subscription is managed by the org. But perhaps still a good idea to provide users with a clear way to switch from their individual subscription in the future. Thanks! 👍🏻
